kensu/exp/__init__.py

```diff
@@ -0,0 +1,293 @@
+import logging
+
+import urllib3
+urllib3.disable_warnings()
+from kensu.utils.kensu import Kensu
+from kensu.client.models import *
+from kensu.utils.helpers import to_datasource
+
+from pyspark.sql import DataFrame
+from pyspark.sql import SparkSession
+from kensu.pyspark import get_process_run_info, get_inputs_lineage_fn, register_manual_lineage, get_spark_from_df, \
+    get_df_with_inmem_tag
+from kensu.utils.kensu_provider import KensuProvider
+
+
+def get_spark_session():
```

Comment:
ideally we shouldn't use it:
- maybe slow
- what if there are multiple SparkSession objects (e.g. databricks, old and new spark session, some spark internals)?